src/main/java/org/folio/dew/batch/acquisitions/edifact/jobs/SaveToMinioTasklet.java

```diff
@@ -52,42 +51,33 @@ public class SaveToMinioTasklet implements Tasklet {
   @SneakyThrows
   public RepeatStatus execute(StepContribution contribution, ChunkContext chunkContext) {
     // retrieve parameters from job context
-    var stepExecution = chunkContext.getStepContext().getStepExecution();
     var jobParameters = chunkContext.getStepContext().getJobParameters();
-    var ediExportConfig = ediObjectMapper.readValue((String)jobParameters.get(EDIFACT_ORDERS_EXPORT), VendorEdiOrdersExportConfig.class);
-    var edifactOrderAsString = (String) ExecutionContextUtils.getExecutionVariable(stepExecution,"edifactOrderAsString");
+    var ediExportConfig = ediObjectMapper.readValue((String) jobParameters.get(EDIFACT_ORDERS_EXPORT), VendorEdiOrdersExportConfig.class);
+    if (ediExportConfig.getIntegrationType() != ORDERING && ediExportConfig.getTransmissionMethod() != FILE_DOWNLOAD) {
```

Comment:
to minio we should save in any case for both transition methods: File download, and FTP.
We will upload that file into Minio and after this will transfer to Ftp if transitionMethod is Ftp.
But it is required to have file in Minio in any case

Reply:
Okay, will modify, but we still can probably skip the step to download it from minio for FTP upload and just read it from the memory